internal/app/machined/pkg/controllers/files/etcfile_test.go

```diff
@@ -153,11 +153,12 @@ func (suite *EtcFileSuite) TearDownTest() {
 	suite.ctxCancel()
 
 	suite.wg.Wait()
-
```

Comment:
these are not needed anymore?

Reply:
in fact I made `go test -v ./...` pass on Talos repo without root - it skips the tests which are not applicable/can't run without root or some tools.